Safewaan/HonkedIn
# Styling for Create Event, Create Forum Pages

## Trello ticket link

[Developer Task: Implement UI/UX feedback to the forms](https://trello.com/c/EVvyVCIw/125-developer-task-implement-ui-ux-feedback-to-the-forms )



## Implementation description
* Add Styling for the create event page
* Add Styling for the create forum page



## Steps to test
1. Verify an event can be created
2. Verify error messages appear when required text fields are not given
3. Verify a forum can be created
4. Verify error messages appear when required text fields are not given
